Resources/doc/configuration.rst

```diff
@@ -95,7 +95,7 @@ Configuration Reference
                         platform_service:     ~
                         auto_commit:          ~
 
-                        # If set to "/^sf2_/" all tables not prefixed with "sf2_" will be ignored by the schema
+                        # If set to "/^sf2_/" all tables, and any named objects such as sequences, not prefixed with "sf2_" will be ignored by the schema
```

Comment:
Please wrap the comment consistently with the previous version

Reply:
WDYM? In both versions it's single line so no wrapping, isn't it?

Reply:
No, the end of the sentence is on a new line (`tool.` is on line 99), so there is wrapping. The goal here is to avoid horizontal in the docs, let's not make the docs harder to read.